scripts/main.js

```diff
@@ -0,0 +1,70 @@
+import { hotels } from "../scripts/db.js";
+const logger = (fun) => console.log(fun);
+
+// -------  HOMEWORK 6 (Array) ---------
+// ------------- part 5 ----------------
+
+export const palindrome = (word) => (word === word.split("").reverse().join("") ? 1 : 0);
```

Comment:
The return format doesn't assign the task.

Reply:
In this task i should replace 1 on "true" and 0 on "false"?